kommandir/inventory/peons

```diff
@@ -16,6 +17,7 @@ rhel7-2-docker-latest
 
 rhelatomic7-4-docker
 rhelatomic7-4-docker-latest
+rhelatomic7-4-docker-latest-overlay2
```

Comment:
I don't know if this is possible: by the time rhelatomic comes up, docker has started and has configured its storage.